modules/kernel/src/main/scala/tofu/Wander.scala

```diff
@@ -0,0 +1,18 @@
+package tofu
+
+import cats.Traverse
+import tofu.internal.carriers.{WanderCarrierCE2, WanderCarrierCE3}
+
+trait Wander[F[_]] {
```

Comment:
Monix has deprecated the name, and so should we. 
I believe it will confuse newcomers more, as Monix itself as a library is not that popular anymore.

Reply:
Like we discussed, it's not essential to name it Wander, but I need to have specific proposals for a new name, otherwise I'll keep this one